src/status_im/ui/screens/wallet/request/events.cljs

```diff
@@ -32,3 +32,8 @@
       {:db (-> db
                (assoc-in [:wallet/request-transaction :amount] (money/ether->wei value))
                (assoc-in [:wallet/request-transaction :amount-error] error))})))
+
+(handlers/register-handler-fx
+  :wallet.request/set-symbol
+  (fn [{:keys [db]} [_ symbol]]
```

Comment:
Can we use a diff variable name than symbol? Builtin Clojure name

Reply:
:+1: